### PR TITLE
fixes parse_nmrstar_ast usability (Python 3.6)

### PR DESCRIPTION
**Pull request by puinenveturi**
Hi @mattfenwick,

I have done the necessary fixes to bring my "test_import.py" (see committed test file) to work. Mostly just Python3 compatibility, for example maps are lazily evaluated in Python3, so you have to force evaluate them by converting them to lists. 

Please double check the "nmrpystar/test/testparser.py" modifications, I was able to pass the tests with these, but I am not sure if it is a valid move to edit these tests.

If you are agree to porting the library to Python3 I would be happy to help.

Best regards,
Daniel

### CODE REVIEW

File: nmrpystar/test/testparser.py [hunk 1]
```diff
@@ -40,7 +40,7 @@ class TestParserErrors(u.TestCase):
     def test_cst_problem(self):
         self.assertEqual(parse_nmrstar_ast("data_hi save_me # oop \n "), 
                          bad({'phase': 'CST construction', 
-                              'message': [('data', (1,1)), ('save', (1,9)), ('save close', 'EOF')]}))
+                              'message': [('data', 'EOF'), ('save', 'EOF'), ('save close', 'EOF')]}))
```
Comment (derPuntigamer) [inline nmrpystar/test/testparser.py]: @mattfenwick this was possibly not a valid move

**Comment by mattfenwick** [comment]
Hi @puinenveturi thanks for taking a look at this!  so sorry that it took me forever to see this ... 

will take a look and get back to you !!

File: nmrpystar/test/testparser.py (hunk 1 above)
Comment (mattfenwick) [reply nmrpystar/test/testparser.py:43]: ooh yeah off the top of my head I'm not sure if this makes sense or not, overall I'm fine with it if it's consistent with everything else and passes the tests, but maybe it should be split into a separate PR?

**Comment by puinenveturi** [comment]
It seems to be solved now, my test import script works with the new master branch.

**Pull request closed without merging**